SPECS/libreswan.spec

```diff
@@ -199,26 +218,149 @@ bunzip2 *.fax.bz2
 %attr(0700,root,root) %dir %{_sysconfdir}/ipsec.d/policies
 %attr(0644,root,root) %config(noreplace) %{_sysconfdir}/ipsec.d/policies/*
 %attr(0644,root,root) %config(noreplace) %{_sysconfdir}/sysctl.d/50-libreswan.conf
-%attr(0700,root,root) %dir %{_localstatedir}/log/pluto
-%attr(0700,root,root) %dir %{_localstatedir}/log/pluto/peer
 %attr(0755,root,root) %dir %{_rundir}/pluto
 %attr(0644,root,root) %{_tmpfilesdir}/libreswan.conf
 %attr(0644,root,root) %{_unitdir}/ipsec.service
 %attr(0644,root,root) %config(noreplace) %{_sysconfdir}/pam.d/pluto
+%config(noreplace) %{_sysconfdir}/logrotate.d/libreswan
 %{_sbindir}/ipsec
 %{_libexecdir}/ipsec
 %attr(0644,root,root) %doc %{_mandir}/*/*
-%{_libdir}/fipscheck/pluto.hmac
 
 %changelog
-* Fri Sep 30 2022 Samuel Verschelde <stormi-xcp@ylix.fr> - 3.26-1.3
-- Rebuild for XCP-ng 8.3 alpha
-
-* Thu Jul 02 2020 Samuel Verschelde <stormi-xcp@ylix.fr> - 3.26-1.2
-- Rebuild for XCP-ng 8.2
-
-* Fri Dec 20 2019 Samuel Verschelde <stormi-xcp@ylix.fr> - 3.26-1.1
-- Rebuild for XCP-ng 8.1
+* Wed Jun 26 2024 David Morel <david.morel@vates.tech> - 4.12-2.1
```

Comment:
looks strange to have 4.12-2.1 after 4.12-2.3 ?

Reply:
Fixed